src/cargo/util/config.rs

```diff
@@ -19,21 +18,19 @@ use self::ConfigValue as CV;
 pub struct Config<'a> {
     home_path: Path,
     shell: RefCell<&'a mut MultiShell>,
-    jobs: u32,
-    target: Option<string::String>,
-    rustc_version: string::String,
+    rustc_version: String,
     /// The current host and default target of rustc
-    rustc_host: string::String,
+    rustc_host: String,
+    values: RefCell<HashMap<String, ConfigValue>>,
+    values_loaded: Cell<bool>,
```

Comment:
I wanted the `values` function below to return `Ref<HashMap>` instead of `Ref<Option<...>>` so it would be a little more usable from a consumer perspective (and you can't `.map` a `Ref`)